keyboards/handwired/oem_ansi_fullsize/oem_ansi_fullsize.h

```diff
@@ -0,0 +1,72 @@
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+/* Matrix col/row mapping
+ *
+ *                                                          10     11   12   13   14     15   16   17     18   19   20   21
+ *          C0   C1   C2   C3   C4   C5  C6 C7   C8   C9   CA     CB   CC   CD   CE     CF   CG   CH     CI   CJ   CK   CL
+ *         ┌───┐     ┌───┐┌───┐┌───┐┌───┐  ┌───┐┌───┐┌───┐┌───┐  ┌───┐┌───┐┌───┐┌───┐  ┌───┐┌───┐┌───┐
+ *     R0  │K00│     │K02││K03││K04││K05│  │K07││K08││K09││K0A│  │K0B││K0C││K0D││K0E│  │K0F││K0G││K0H│
+ *         └───┘     └───┘└───┘└───┘└───┘  └───┘└───┘└───┘└───┘  └───┘└───┘└───┘└───┘  └───┘└───┘└───┘
+ *
+ *         ┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───────┐  ┌───┐┌───┐┌───┐  ┌───┐┌───┐┌───┐┌───┐
+ *     R1  │K10││K11││K12││K13││K14││K15││K16││K17││K18││K19││K1A││K1B││K1C││  K1E  │  │K1F││K1G││K1H│  │K1I││K1J││K1K││K1L│
+ *         └───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───────┘  └───┘└───┘└───┘  └───┘└───┘└───┘└───┘
+ *         ┌─────┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌─────┐  ┌───┐┌───┐┌───┐  ┌───┐┌───┐┌───┐┌───┐
+ *     R2  │ K20 ││K22││K23││K24││K25││K26││K27││K28││K29││K2A││K2B││K2C││K2D││ K2E │  │K2F││K2G││K2H│  │K2I││K2J││K2K││   │
+ *         └─────┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└─────┘  └───┘└───┘└───┘  └───┘└───┘└───┘│K2L│
+ *         ┌───────┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌────────┐                   ┌───┐┌───┐┌───┐│   │
+ *     R3  │  K30  ││K32││K33││K34││K35││K36││K37││K38││K39││K3A││K3B││K3C││  K3D   │                   │K3I││K3J││K3K││   │
+ *         └───────┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└────────┘                   └───┘└───┘└───┘└───┘
+ *         ┌─────────┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───┐┌───────────┐       ┌───┐       ┌───┐┌───┐┌───┐┌───┐
+ *     R4  │   K41   ││K42││K43││K44││K45││K46││K47││K48││K49││K4A││K4B││    K4D    │       │K4G│       │K4I││K4J││K4K││   │
+ *         └─────────┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───┘└───────────┘       └───┘       └───┘└───┘└───┘│K4L│
+ *         ┌────┐┌────┐┌────┐┌──────────────────────────────┐┌────┐┌────┐┌────┐┌────┐  ┌───┐┌───┐┌───┐  ┌────────┐┌───┐│   │
+ *     R5  │K50 ││K51 ││K53 ││               K56            ││K5A ││K5B ││K5D ││K5E │  │K5F││K5G││K5H│  │   K5I  ││K5K││   │
+ *         └────┘└────┘└────┘└──────────────────────────────┘└────┘└────┘└────┘└────┘  └───┘└───┘└───┘  └────────┘└───┘└───┘
+ *
+ */
+
+// clang-format off
+#define LAYOUT( \
```

Comment:
```suggestion
#define LAYOUT_fullsize_ansi( \
```

Reply:
Done, but with `LAYOUT_ansi_fullsize` for consistency with the name of the board.